Dynatrace/dynatrace-operator
# Change logMonitoring error handling in case of no kubernetesClusterMEID

## Description

https://dt-rnd.atlassian.net/browse/DAQ-3138

In case we have no `kuberentesClusterMEID` and kubernetes monitoring is not enabled, we error with a stacktrace, which pollutes the logs a lot.

With this I changed the way we behave: We only log that the missing `kubernetesClusterMEID` is the problem and and then return nil, as this is expected to not work.



## How can this be tested?

Deploy operator, make sure that your cluster has no `kubernetesClusterMEID`, enable `logMonitoring` and **no** kubernetes monitoring. Have a look at the dk status and the logs
